lib/rxjs/rxjs.ts

```diff
@@ -145,26 +145,28 @@ import {Observable, Subscriber, Subscription} from 'rxjs';
     Subscriber.prototype.next = function() {
       const currentZone = Zone.current;
       const subscriptionZone = this._zone;
+      const args: [any?] = Array.prototype.slice.call(arguments) as [any?];
```

Comment:
This seems like extra work which we have to do. Why is the `slice` needed? Can't we just cast it somehow to make TS happy?

Reply:
Sure,  without making a new `slice`, just `cast arguments to any`.